src/operator/api/v1alpha3/clientintents_types.go

```diff
@@ -155,6 +157,9 @@ type Intent struct {
 	//+optional
 	Type IntentType `json:"type,omitempty" yaml:"type,omitempty"`
 
+	//+optional
+	Port int32 `json:"port,omitempty" yaml:"port,omitempty"`
```

Comment:
We still need to figure out a way to do without specifying the port manually - let's try to figure this out when we talk